pages/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: FAQ
+permalink: /FAQ/
+nav_page: whatWeDo
+nav_page_specific: faq
+---
+
+# Frequently Asked Questions
+
+<hr class="bg-primary"/>
+
+Have a question about AB Tech, getting involved, or hiring us? We have answers (well, most of the time). Don't see your question below? Please email <a href = "mailto: abtech@andrew.cmu.edu">abtech@andrew.cmu.edu</a> with any questions you may have.
```

Comment:
Change link to be `<a href="mailto:abtech@andrew.cmu.edu" class="user-select-all">abtech@andrew.cmu.edu</a>`. This adds a helper class that makes selecting it a bit easier.